pkg/lightning/backend/local/local.go

```diff
@@ -208,8 +237,13 @@ func (e *File) getSizeProperties() (*sizeProperties, error) {
 	return sizeProps, nil
 }
 
+func isStateLocked(state importMutexState) bool {
+	return state&(importMutexStateClose|importMutexStateImport) != 0
+}
+
 func (e *File) isLocked() bool {
-	return e.isImportingAtomic.Load() != 0
+	// the engine is locked only in import or close state.
+	return isStateLocked(importMutexState(e.isImportingAtomic.Load()))
```

Comment:
@kennytm PTAL. I'm not sure if we need to treat the `ReadLock` state (flush state) as locked or unlocked

Reply:
isLocked is used to determine if an engine has already been importing to TiKV when computing disk quota. It should return false when TiKV is not involved.